backend/crossref.py

```diff
@@ -558,9 +559,9 @@ def fetch_and_save_new_records(self, starting_cursor='*', batch_time=datetime.ti
                     p = Paper.from_bare(bare_paper)
                     p.update_index()
                 except ValueError as e:
-                    print((record.get('DOI') or 'unknown DOI') + ': '+str(e))
+                    logger.info(record.get('DOI', 'unkown DOI') + ': %s' % e)
```

Comment:
What is the decision with regards to having `exc_info` or exception text added in the string? (see `exc_info` above)

Reply:
Or `logger.exception` :)

Reply:
I think `info` should be right here in terms of severity level. This is something that routinely happens when fetching bogus metadata from crossref (papers with no authors, papers with the abstract in the title field, and so on).

Reply:
The above construction just gives the exception message but not the traceback / exc_info, i.e. looks as before:
`2019-03-22 10:01:54 INFO dissemin.backend.crossref:562  10.1007/s35152-018-0026-3: Invalid author`
I think: If this is reported, there should be reason, i.e. what is wrong with the metadata of the DOI.

Reply:
Ok!